cmd/v0.22/pingsource-cleanup/README.md

```diff
@@ -0,0 +1,82 @@
+# PingSource cleanup post-0.22.0 install
+
+The following is a log of testing commands to try out this job.
+
+The job's log is the yaml of all resources that were deleted. The job uses
+eventing's service account.
+
+---
+
+## kind start cluster:
+
+```
+cat <<EOF | kind create cluster --name pingsource --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  image: kindest/node:v1.16.9@sha256:7175872357bc85847ec4b1aba46ed1d12fa054c83ac7a8a11f5c268957fd5765
+- role: worker
+  image: kindest/node:v1.16.9@sha256:7175872357bc85847ec4b1aba46ed1d12fa054c83ac7a8a11f5c268957fd5765
+EOF
+```
+
+## install knative eventing 0.16:
+
+```
+kubectl apply --filename https://github.com/knative/eventing/releases/download/v0.16.0/eventing-crds.yaml
```

Comment:
why .16? we don't even support it anymore. Cut&Paste?

Reply:
@vaikas yeah...wait…… I just reuse the last job file, which haven't  been put into practice yet, so l add a hold label